sgx/report-cache/untrusted/src/lib.rs

```diff
@@ -187,42 +187,40 @@ impl<E: ReportableEnclave, R: RaClient> ReportCache<E, R> {
             "Starting enclave report cache update process..."
```

Comment:
This is actually a comment potentially outside the scope of this PR, but while we're in here, could we possibly add an info-level log message here that prints out the ias report (the current one is trace in the ra_client, [here](https://github.com/mobilecoinofficial/mobilecoin/blob/master/attest/net/src/ias.rs#L132), and also add the MRENCLAVE and MRSIGNER values? I think we've been wanting to have those prominently in the logs. Otherwise we can make a ticket for that for another PR.

Reply:
This PR is already pretty big, and there will be work to port internal to the new verifier APIs as well, so I'd rather just JIRA that for later.

Reply:
👍

Reply:
https://mobilecoin.atlassian.net/browse/MCC-1717